### PR TITLE
SDL: set WM_CLASS on Wayland/X11

### PR DESCRIPTION
**Pull request by pazos**
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9280)

### CODE REVIEW

File: frontend/device/sdl/device.lua [hunk 1]
```diff
@@ -2,6 +2,14 @@ local Event = require("ui/event")
 local Generic = require("device/generic/device")
 local SDL = require("ffi/SDL2_0")
 local logger = require("logger")
+local ffi = require("ffi")
```
Comment (Frenzie) [inline frontend/device/sdl/device.lua:5]: Not quite sure where it should go really due to what it is, but I'd say either at the very top or above logger, definitely not there. 😁

**Review by Frenzie — APPROVED** [review]
Otherwise fine by me for obvious reasons.

**Comment by pazos** [comment]
ups, sorry.

Didn't notice you requested @NiLuJe review. I was under the impression it was autorequested as a code keeper :p

**Comment by Frenzie** [comment]
I'm expecting agreement but there might be something I overlooked. In case of the latter a small follow up doesn't hurt. :-)

**Comment by NiLuJe** [comment]
I'm woefully unfamiliar with the SDL specifics, but that seems sensible & harmless enough ;).

**Comment by Frenzie** [comment]
The SDL part is correct in any case, no doubt about that. ;-) (Cf. SDL source.)

**Pull request merged**